### PR TITLE
improved Readme

### LINKED ISSUE #70: Improve README
Right now readme doesn't display any code example to display the beauty of abell. That can mean potentially loss in conversion of users. 

We should improve the readme.

### PR DESCRIPTION
**Pull request by gurrrung**
closes #70 

## What type of PR is this?

- [ ] Feature
- [ ] Bug Fix
- [ ] Refactor
- [x] Documentation Update

## Added unit tests?

- [ ] No, I want help in writing tests.
- [ ] No, I want someone else to write tests.
- [x] No, The addition does not need tests.
- [ ] Yes

## Random GIF to +1 someone's day
tenor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,24 @@ npm run dev
 
 Check out https://abelljs.org for complete documentation.
 
+## 🚀 &nbsp; Usage
```
Comment (saurabhdaware) [inline README.md:24]: Maybe "Example" or "Hello World" is a better headline here. lmk what you think

**Review by saurabhdaware — CHANGES_REQUESTED** [review]
Thank you so much @gurrrung. This makes sense. I've requested tiny change below. Let me know if it makes sense

File: README.md (hunk 1 above)
Comment (gurrrung) [reply README.md:24]: `Example` sounds good to me.

**Review by pantharshit00 — APPROVED** [review]

**Review by saurabhdaware — APPROVED** [review]
Thank you so much @gurrrung 🥳🥳

**Pull request merged**